app/connections/clean_connection.rb

```diff
@@ -0,0 +1,23 @@
+# This stands in for ActiveFedora::CleanConnection. It behaves the same way,
```

Comment:
Why does `ActiveFedora::CleanConnection` need to clear `hasModel`? Looking at https://github.com/projecthydra/active_fedora/commit/dffef733b0d9e952972353edf9f69375ad1bc57e, it seems like it was added to support this feature, so it's a little funny we're actively opting out.